packages/create-toolpad-app/src/templates/auth/envLocal.ts

```diff
@@ -0,0 +1,22 @@
+import { ProvidersTemplate } from '../../types';
+
+const env: ProvidersTemplate = (providers) => {
+  const oAuthProviders = providers.filter((provider) => provider !== 'Credentials');
+  return `
+# Generate a secret with \`npx auth secret\`
+# and replace the value below with it
+AUTH_SECRET=secret
+
+# Add the CLIENT_ID and CLIENT_SECRET from your OAuth provider
+# to the .env.local file
+${oAuthProviders
+  .map(
+    (provider) => `
+${provider?.toUpperCase()}_CLIENT_ID=
```

Comment:
Only null-coalesce things when it makes sense. This one can't be null according to the types. And, if it could be `null` for some reason, that would result in `"null_CLIENT_ID"` which would be a bug anyway.


```suggestion
${provider.toUpperCase()}_CLIENT_ID=
```

Besides, you'd probably want to convert to constant case, to avoid `"azure-devops"` from turning into `"AZURE-DEVOPS"`

```tsx
// This function can go in @toolpad/utils/strings
function kebabToConstantCase(input: string): string {
  return input.split('-').map(part => part.toUpperCase()).join('_')
}
```